### DISCUSSION TITLE
Task1 奖励发放🎉

### DISCUSSION THREAD

**Discussion by phoouze** [openbuildxyz/mina-zkapp-bootcamp#225] [Announcements]
[Task1](https://github.com/openbuildxyz/mina-zkapp-bootcamp/blob/main/task/task1.md)  的奖励已发放，请查看[奖励统计表](https://github.com/openbuildxyz/mina-zkapp-bootcamp/blob/main/reward/task1.md)🎉

没有收到奖励的同学请：
- 查看自己的 pr 是否被合并，有问题的 pr 请及时处理。
- 查看自己是否在 **learn/username/username.md** 中提交钱包地址，未提交钱包地址的请尽快提交。
- 是否按照[规则](https://github.com/openbuildxyz/mina-zkapp-bootcamp#%E4%BB%BB%E5%8A%A1%E6%8F%90%E4%BA%A4)提交。
- 只有在 Task1 任务时间内(11月25日前)提交才有奖励， 请确认。

请检查自己的提交内容和 pr，及时更新。漏发的同学会进行补发，有任何问题请在本讨论区进行讨论。

**Comment by wangyuanzcm** [comment]
上周已经提交pr，今天提示有我有问题，再次merge之后是不是就没有奖励了？我看今天的更新的pr没有被合并

**Reply by phoouze** [reply]
已经合并了， 只要在task时间内提交的都会算； 奖励已发，注意查收

**Comment by geyee** [comment]
提交合并了，不知是否是md换行等原因导致未能提取钱包地址。没有收到奖励。

**Reply by phoouze** [reply]
已发 , https://minascan.io/mainnet/tx/5JuYjNQdoz7qnTW15EUYynjyTGXX5SbQx1FVyv9oRk1dM2vsq8wW

**Comment by fanofmusk** [comment]
因为tx hash填错了，已重新提交了pr，麻烦看下是否已合并，目前还没收到奖励

**Reply by phoouze** [reply]
今天的提交比较少，明天会统一处理

**Reply by phoouze** [reply]
已发，https://minascan.io/mainnet/tx/5JukXaVBvGAeaumEYGjYHS6cPnrvJofdHvQb5AMKqWtHSHgE84kW

**Comment by betazhao** [comment]
你好，task1提交了，没有收到奖励

**Reply by phoouze** [reply]
今天会统一处理

**Reply by phoouze** [reply]
https://github.com/openbuildxyz/mina-zkapp-bootcamp/pull/179 你的提交只是在github上报名了，没有按参与要求完成task1

**Reply by betazhao** [reply]
https://github.com/openbuildxyz/mina-zkapp-bootcamp/pull/179这个是我提交的pr

**Comment by Tsuizen** [comment]
哈喽，task1的pr已经被合并了，没有收到奖励

**Reply by phoouze** [reply]
今天会统一处理

**Reply by phoouze** [reply]
已发，https://minascan.io/mainnet/tx/5JtjMFJvymZdcGPRQxPf943VWxZMnK1tDzEQutuuTj3S6HcfHXBS
